.github/workflows/charmcraft-pack.yaml

```diff
@@ -0,0 +1,38 @@
+name: Charmcraft Pack Test
+
+on: [push, pull_request, workflow_call]
+
+jobs:
+  charmcraft-pack:
+    runs-on: ubuntu-20.04
+
+    steps:
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: "3.8"
+
+    - name: Checkout test charm repository
+      uses: actions/checkout@v2
+      with:
+        repository: jnsgruk/hello-kubecon
+
+    - name: Update 'ops' dependency in test charm to latest
+      run: |
+        # We need to reference the PR branch's repo and commit (not the GITHUB_SHA
+        # temporary merge commit), because charmcraft pack does a git checkout which
+        # can't see the temporary merge commit.
+        sed -i -e "/^ops[ ><=]/d" -e "/canonical\/operator/d" -e "/#egg=ops/d" requirements.txt
+        echo -e "\ngit+${{ github.event.pull_request.head.repo.clone_url }}@${{ github.event.pull_request.head.sha }}#egg=ops" >> requirements.txt
+        cat requirements.txt
+
+    - name: Install charmcraft
+      run: |
+          sudo snap refresh lxd --channel=latest
```

Comment:
If we do move to jammy, might be best to use the canonical/setup-lxd action to avoid firewall issues

Reply:
You can specify the version you want with the action, it'll just take care of initialising it and deconflicting firewall rules for you 😀

Reply:
I've updated to jammy and pinned lxd to 5.0 to be safe.